docs/dashboards/kpi-component.md

```diff
@@ -0,0 +1,96 @@
+KPI Component
+===
+
+The KPIComponent allows you to visualize key performance indicators.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/component-kpi allow="fullscreen"></iframe>
+
+## How to start
+1. The KPIComponent is included in the Dashboard package, so nothing apart from the basic dashboards package needs to be imported.
+
+2. Define a cell using a unique identifier for example cell: `dashboard-col-0`.
```

Comment:
```suggestion
2. Define a cell using a unique identifier for example `cell: dashboard-col-0`.
```

Reply:
Missing quotes, so I will commit it on my own.